python/paddle/distributed/auto_parallel/tuner/profiler.py

```diff
@@ -0,0 +1,287 @@
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
Can we make this profiler function into the cost model? or make this available to engine api or as a basic feature for auto parallel.

Reply:
will be considered